workitem/event/event_repository.go

```diff
@@ -0,0 +1,161 @@
+package event
+
+import (
+	"context"
+	"strings"
+
+	"github.com/jinzhu/gorm"
+	errs "github.com/pkg/errors"
+	uuid "github.com/satori/go.uuid"
+
+	"github.com/fabric8-services/fabric8-wit/account"
+	"github.com/fabric8-services/fabric8-wit/workitem"
+)
+
+// event types
+const (
+	APIStringTypeEvents = "events"
+
+	Assignees = "assignees"
+	State     = "state"
+	Labels    = "labels"
+	Iteration = "iteration"
+)
+
+// EventNameMap maps system key to a normal string
+var EventNameMap = map[string]string{
```

Comment:
Why map the name to something else? A Field already has a name?

Reply:
Address here. 2417c0cfcf55c9f8436f8eb73be675f6733ee6f0